hypershift-operator/controllers/hostedcluster/hostedcluster_webhook.go

```diff
@@ -75,28 +103,33 @@ func validateNetworkCIDRs(hc *hyperv1.HostedCluster) field.ErrorList {
 	serviceCIDR := hc.Spec.Networking.ServiceCIDR
 	machineCIDR := hc.Spec.Networking.MachineCIDR
 
-	// Validate CIDR format..
+	// I think if these are all unset we should ignore them.
+	if podCIDR == "" && serviceCIDR == "" && machineCIDR == "" {
```

Comment:
Did you observe that happening?

Reply:
In unit tests only.. I could make it so they are always set.  Also I wondered about shifting to the new API? I saw somewhere else in the code I think they are being set, but the new API uses slices so you can have multiple CIDRs in each.. I'm guessing this would be deprecated at some point?